lib/wwt.js

```diff
@@ -35,6 +39,18 @@ var WWTView = widgets.DOMWidgetView.extend({
 
         WWTView.__super__.initialize.apply(this, arguments);
 
+        var self = this;
+        setInterval(function () { self.update_view_data(); }, 100);
```

Comment:
If I'm reading this correctly, we set up to call `update_view_data()` periodically every 100 ms, right? That's not the end of the world but it's a bit less than ideal — is this definitely necessary? If so, a comment in the code explaining why would be helpful.

Reply:
As per offline discussion with @astrofrog, we're OK with this approach for now. #210 will track the broader issue and potential solutions.